docs/en/sql_reference/statements/create.md

```diff
@@ -304,4 +304,179 @@ Depending on dictionary [layout](../../sql_reference/dictionaries/external_dicti
+
+```sql
+SET ROLE accountant;
+SELECT * FROM db.*;
+```
+
+## CREATE ROW POLICY {#create-row-policy-statement}
+
+Creates a filter for rows, which a user can read from a table. Also, you can create row filters in the [user settings](../../operations/settings/settings_users.md#user-databases-settings).
+
+### Syntax {#create-row-policy-syntax}
+
+``` sql
+CREATE [ROW] POLICY [IF NOT EXISTS | OR REPLACE] policy_name  [ON CLUSTER] ON [db.]table
+    [AS {PERMISSIVE | RESTRICTIVE}]
+    [FOR SELECT]
+    [USING condition]
+    [TO {role [,...] | ALL | ALL EXCEPT role [,...]}]
+```
+
+#### Section AS {#create-row-policy-as}
+
+Using this section you can create permissive or restrictive policies.
+
+Permissive policy grants access to rows. Permissive policies which apply to the same table are combined together using the boolean `OR` operator. Policies are permissive by default.
+
+Restrictive policy restricts access to row. Restrictive policies which apply to the same table are combined together using the boolean `AND` operator.
+
+Restrictive policies apply to rows that passed the permissive filters. If you set restrictive policies but no permissive policies, the user can't get any row from the table.
+
+#### Section TO {#create-row-policy-to}
+
+In the section `TO` you can give a mixed list of roles and users, for example, `CREATE ROW POLICY ... TO accountant, vasya@localhost`.
+
+Keyword `ALL` means all the ClickHouse users including current user. Keywords `ALL EXCEPT` allow to to exclude some users from the all users list, for example `CREATE ROW POLICY ... TO ALL EXCEPT accountant, vasya@localhost`
+
+### Examples
+
+- `CREATE ROW POLICY filter ON mydb.mytable FOR SELECT USING a<1000 TO accountant, vasya@localhost`
+- `CREATE ROW POLICY filter ON mydb.mytable FOR SELECT USING a<1000 TO ALL EXCEPT mira`
+
+
+## CREATE QUOTA {#create-quota-statement}
+
+Creates a [quota](../../operations/quotas.md) that can be assigned to a user of role.
+
+### Syntax {create-quota-syntax}
+
+``` sql
+CREATE QUOTA [IF NOT EXISTS | OR REPLACE] name [ON CLUSTER]
+    [KEYED BY {'none' | 'user name' | 'ip address' | 'client key' | 'client key or user name' | 'client key or ip address'}]
+    [FOR [RANDOMIZED] INTERVAL number {SECOND | MINUTE | HOUR | DAY}
+        {[SET] MAX { {QUERIES | ERRORS | RESULT ROWS | RESULT BYTES | READ ROWS | READ BYTES | EXECUTION TIME} = {number | ANY} } [,...] | [SET] TRACKING} [,...]]
+    [TO {role [,...] | ALL | ALL EXCEPT role [,...]}]
+```
+
+
+## CREATE SETTINGS PROFILE {#create-settings-profile-statement}
+
+Creates a [settings profile](../../operations/settings/settings_profiles.md) that can be assigned to a user of role.
+
+### Syntax {create-quota-syntax}
+
+``` sql
+CREATE SETTINGS PROFILE [IF NOT EXISTS | OR REPLACE] name [ON CLUSTER]
+    [SETTINGS variable [= value] [MIN [=] min_value] [MAX [=] max_value] [READONLY|WRITABLE] | PROFILE 'profile_name'] [,...]
+```
```

Comment:
I suppose an example here would be useful. See https://github.com/ClickHouse/ClickHouse/blob/1de80d61e94fc17e418292a12fe0cf459a40b56f/tests/integration/test_settings_profile/test.py